Library/Formula/thc-pptp-bruter.rb

```diff
@@ -1,14 +1,12 @@
-require 'formula'
-
 class ThcPptpBruter < Formula
-  homepage 'http://thc.org'
-  url 'http://freeworld.thc.org/releases/thc-pptp-bruter-0.1.4.tar.gz'
-  sha1 '88a797ed0dcfb79aba92b319e29d1c5d8c1b14a4'
+  homepage "https://thc.org"
+  url "https://freeworld.thc.org/releases/thc-pptp-bruter-0.1.4.tar.gz"
+  sha1 "88a797ed0dcfb79aba92b319e29d1c5d8c1b14a4"
 
   def install
     # The function openpty() is defined in pty.h on Linux, but in util.h on OS X.
-    # See http://groups.google.com/group/sage-devel/msg/97916255b631e3e5
-    inreplace 'src/pptp_bruter.c', 'pty.h', 'util.h'
+    # See https://groups.google.com/group/sage-devel/msg/97916255b631e3e5
+    inreplace "src/pptp_bruter.c", "pty.h", "util.h"
 
     system "./configure", "--prefix=#{prefix}"
     system "make"
```

Comment:
For a test I'd need to know what this package does, which I don't, I'm sorry.

As for the `brew audit`, I'm still wrestling with an error:
```
Error: The 'rubocop' gem is installed but couldn't find 'rubocop' in the PATH:
```
The PATH added by rubygems is 2.0.0 (system ruby), while I'm using 2.2.0 from Homebrew. Still in search for the solution...